buddyboss/buddyboss-platform
# PROD-793 Fix/2540 Group messages notification conflict with media uploading components

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2540.

### How to test the changes in this Pull Request:

1. Deactivate the media uploading component.
2. Go to any group and sent messages to the group.
3. Switch back to any member and check the notification.

### Proof Screenshots or Video
http://somup.com/crQjfUrOLn
http://image.prntscr.com/image/tDYZyLb-SZW96_Mv7LtVfA.png

final:
http://somup.com/crQj1frO9M

### Other information:




### Changelog entry

> Moved & renamed hook function to "bb_messages_save_group_data()" from the Media component folder - bp-media-filters.php file to Message component folder - bp-messages-filters.php file and deprecated the old "bp_media_messages_save_group_data()" in bp-media-filters.php file
